src/mcts/search.cc

```diff
@@ -125,16 +125,17 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
     ++multipv;
     uci_infos.emplace_back(common_info);
     auto& uci_info = uci_infos.back();
+    auto score = edge.GetQ(default_q);
     if (score_type == "centipawn") {
-      uci_info.score = 295 * edge.GetQ(default_q) /
-                       (1 - 0.976953126 * std::pow(edge.GetQ(default_q), 14));
+      score = 135 * -std::log(2 / (score + 1) - 1);
```

Comment:
Maybe keep also the current way under some name? `centipawn_2019`, for example.

Reply:
Restored and refactored the current way.